frame/nfts/src/benchmarking.rs

```diff
@@ -146,25 +145,19 @@ benchmarks_instance_pallet! {
 	force_create {
 		let caller: T::AccountId = whitelisted_caller();
 		let caller_lookup = T::Lookup::unlookup(caller.clone());
-	}: _(SystemOrigin::Root, caller_lookup, true)
+	}: _(SystemOrigin::Root, caller_lookup, CollectionConfig::all_settings_enabled())
 	verify {
 		assert_last_event::<T, I>(Event::ForceCreated { collection: T::Helper::collection(0), owner: caller }.into());
 	}
 
 	destroy {
 		let n in 0 .. 1_000;
-		let m in 0 .. 1_000;
```

Comment:
In the weight file it looks like these components are used. Why did you remove them?

Reply:
Because previously it was possible to create metadata and attributes for non-existing records and that benchmark worked. After I added a check whether the record exists, that benchmark started to fail when we try to add metadata (or an attribute) in separate loop